src/Entity/AdherentMessage/Filter/AudienceFilter.php

```diff
@@ -97,6 +97,15 @@ class AudienceFilter extends AbstractAdherentMessageFilter implements ZoneableEn
      */
     private ?bool $isCommitteeMember = null;
 
+    /**
+     * @ORM\Column(nullable=true)
+     *
+     * @Groups({"adherent_message_update_filter"})
+     *
+     * @Assert\Choice(choices=App\Entity\ElectedRepresentative\MandateTypeEnum::TYPE_CHOICES_CONTACTS, strict=true)
```

Comment:
strict=true is the default value
```suggestion
     * @Assert\Choice(choices=App\Entity\ElectedRepresentative\MandateTypeEnum::TYPE_CHOICES_CONTACTS)
```

Reply:
fixing in #8771